src/js/expander.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("click", event => {
+	if (!event.target.closest(".expander button")) return false;
```

Comment:
It would be more efficient to register the click event listener directly to ".expander button". You may need to register the click event within the `window.load` event listener to make sure the page has been loaded and buttons are available for hooking up the click event.

Reply:
@cindyli Just to clarify, your suggestion is to attach a click event listener to all expander buttons once window has loaded? Thanks!

Reply:
Yes, @greatislander.

Reply:
(I was following this approach, slightly modified to make sure we catch clicks on the `<svg>` within the button: https://gomakethings.com/listening-for-click-events-with-vanilla-javascript/#the-vanilla-js-way-to-listen-for-click-events).

Reply:
This article said: `While you can listen for clicks on specific elements, my recommended approach is to listen for all clicks on the document, and then check if the clicked element has the selector you care about.`

I'm not sure what's the of doing it while specific elements are known and listen to the click event directly. It feels to me the suggested approach is less efficient.

Reply:
@cindyli The author has a follow-up explanation which is interesting: https://gomakethings.com/why-event-delegation-is-a-better-way-to-listen-for-events-in-vanilla-js/

Reply:
It's an interesting argument. I agree this approach helps to write less code but I think it's bad for the performance and the code logic.